gk/main.c

```diff
@@ -1548,25 +1476,8 @@ lookup_fe_from_lpm(struct ipacket *packet, uint32_t ip_flow_hash_val,
 
 	switch (fib->action) {
 	case GK_FWD_GRANTOR: {
-		int ret;
 		struct flow_entry *fe;
 
-		/*
-		 * If the table is full at a given batch,
-		 * the table only has a chance to free entries
-		 * in between batches. So there's no reason to
-		 * risk trying another flow in the current
-		 * batch. However, we will give this flow a
-		 * chance sending a request to the grantor
-		 * server.
-		 */
-		if (instance->has_insertion_failed) {
-			send_request_to_grantor(packet, fib,
-				req_bufs, req_prio, num_reqs,
-				instance, gk_conf);
-			return NULL;
-		}
-
 		/*
 		 * We heuristically drop entries to
 		 * alleviate memory pressure
```

Comment:
We are dropping the heuristics in this pull request, so update this comment.

Reply:
Comment updated.